src/include/restore.h

```diff
@@ -117,6 +117,17 @@ pgmoneta_rollup_backups(int server, char* newest_label, char* oldest_label);
 int
 pgmoneta_extract_incremental_backup(int server, char* label, char** root, char** base);
 
+/**
+ * Get the size of the incremental backup
+ * @param server The server
+ * @param label The label
+ * @param size [out] The size of the incremental backup
+ * @param biggest_file_size [out] The size of the biggest file in the incremental backup
+ * @return 0 on success, 1 if otherwise
+ */
+int
+pgmoneta_backup_size_incremental(int server, char* label, unsigned long* size, uint64_t* biggest_file_size);
```

Comment:
Maybe just pgmoneta_backup_size()... The way you wrote it works on full backup as well.